### DISCUSSION TITLE
Setup if you received a complete board from me

### DISCUSSION THREAD

**Discussion by tjboldt** [tjboldt/Apple2-IO-RPi#64] [Announcements]
## Setup if you received a complete board from me
1. Put in any slot except 0 or 3 (slot 7 preferred as it is the first to boot)
2. Turn on your Apple II
3. Wait for the RPi to start up (will show ... until it connects)
4. If you have in a slot other than 7 and you have other drive controllers, you may need to use `PR#x` where x is which slot you have the card in.
5. After ProDOS boots, type `-RPI.COMMAND`
6. Type `RPI a2wifi list`
7. Type `RPI a2wifi select YOUR_SSID YOUR_WIFI_PASSWORD YOUR_TWO_LETTER_COUNTRY_CODE`
8. Star and Watch this repo on GitHub for the latest updates

**Comment by Abysmal** [comment]
Note : GO may need to be installed, See the setup from scratch discussion.

**Reply by tjboldt** [reply]
Go has for sure been installed on every complete board I sold but since I was running it from an ssh session, it’s likely it’s missing from the path in the environment that the apple2driver process is executing from. Why run from an ssh session instead of the Apple II shell you might ask? Well, the Apple II shell and drive won’t work until the driver is compiled and running on the RPi. So I install everything from a remote session on another computer.

**Comment by Abysmal** [comment]
All good, I actually had fun going through the docs and reinstalling it.. you clean install docs are great!!

**Reply by tjboldt** [reply]
Thanks for testing the full installation instructions. They have changed a few times recently so good to know they worked for you. I may change the dependent library to only require go 1.15 instead of 1.16 since the latest Raspberry Pi OS still only supports 1.15 out of the box. That will make it easier for people getting started and avoid the path issues.

**Comment by DaManDOH** [comment]
Looking to confirm which way the card should be installed: with the Pi connectors towards the back of the Apple II or towards the keyboard. Including pictures of the card installed in the documentation would be very useful.

**Reply by tjboldt** [reply]
![2C7A3D05-4BDE-4EC5-82C1-09501DAB70C5](https://user-images.githubusercontent.com/21348575/174401958-8ebaa54d-415c-4baa-9f1d-5ad6efa66863.jpeg)

The RPi goes to the keyboard end. I can update the docs.

**Reply by DaManDOH** [reply]
Wonderful. Thanks much!

**Comment by JPT77** [comment]
Could you please update the docs for the apple ][, ][+ and ][europlus:
The Apple2-IO-RPi does not boot automatically, you always have to use PR#x (or is this a bug? normal floppy controller in #6 boots just fine.)
Probably do not put the card into Slot 7 as this is reserved for gfx cards (slightly different pinout)
thanks
